media/doc/WINESYNC.txt

```diff
@@ -54,7 +54,7 @@ dll/win32/atl80               # Synced to WineStaging-4.18
 dll/win32/atl100              # Synced to WineStaging-3.3
 dll/win32/avifil32            # Synced to WineStaging-4.18
 dll/win32/avrt                # Synced to Wine-9.7
-dll/win32/bcrypt              # Synced to WineStaging-1.9.23 (+ winetest synced to 8d8936cbb6fea3cac862e059e814527f5361f48b, a.k.a 20161206-BJ)
+dll/win32/bcrypt              # Synced to WineStaging-10.6 commit d9148e48a3257ae80361f02f0bb7a2d607ef18ec (+ winetest synced to d9148e48a3257ae80361f02f0bb7a2d607ef18ec)
```

Comment:
Please remove the commit hash. AFAICT it synced to wine-10.6 and the hash is just confusing, as it implies it is synced to some random version in between.